pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss

```diff
@@ -319,79 +332,6 @@ ul {
     }
   }
 
-  // Lists that have an arrow shape as li backgrounds
```

Comment:
Removing this element class since this section was redesigned to work better with translatability.